suite-native/intl/src/en.ts

```diff
@@ -11,6 +11,16 @@ export const en = {
             confirm: 'Confirm',
             continue: 'Continue',
             next: 'Next',
+            dismiss: 'Dismiss',
+        },
+        unknownError: 'Something went wrong',
+    },
+    messages: {
```

Comment:
The first level of the translation map should reflect the module name.

```suggestion
    messageSystem: {
```

Reply:
9860215781